erpnext/regional/india/e_invoice/einvoice.js

```diff
@@ -99,8 +99,8 @@ erpnext.setup_einvoice_actions = (doctype) => {
 									...data
 								},
 								freeze: true,
-								callback: () => frm.reload_doc() || d.hide(),
-								error: () => d.hide()
+								callback: () => frappe.show_alert({ message:__('E-Way Bill Generated successfully'), indicator:'green'}, 7) && frm.reload_doc() && d.hide(),
+								error: () => frappe.show_alert({ message:__('E-Way Bill was not Generated'), indicator:'red'}, 7)
```

Comment:
```suggestion
								error: () => {
									frappe.show_alert({
										message: __('E-Way Bill was not Generated '),
										indicator:'red'
									}, 7);
									d.hide();
								}
```

Reply:
'E-Way Bill was not Generated ' has trailing whitespace.

Reply:
Can you split this into multiple line manually?

Reply:
done